core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -625,6 +625,8 @@ private static ElevationProvider createElevationProvider(GraphHopperConfig ghCon
             elevationProvider = new SRTMGL1Provider(cacheDirStr);
         } else if (eleProviderStr.equalsIgnoreCase("multi")) {
             elevationProvider = new MultiSourceElevationProvider(cacheDirStr);
+        } else if (eleProviderStr.equalsIgnoreCase("skadi")) {
+            elevationProvider = new SkadiProvider(cacheDirStr);
```

Comment:
Can you add it also to this table? https://github.com/graphhopper/graphhopper/blob/master/NOTICE.md#data